src/emulator/modules/SceGxm/src/SceGxm.cpp

```diff
@@ -18,6 +18,7 @@
 #include <SceGxm/exports.h>
 
 #include "gxm.h"
+#include "util/log.h"
```

Comment:
Come to think of it, rather than using `LOG_` macros directly, there should be a module wrapper around logging, which inserts the function name without the `export_` prefix.

There's some prerequisite work I need to do here in bridging, because I had some difficulty getting the export name to pass through as a template parameter with Visual Studio and just nuked it when I ran out of time during a train journey.